docs/source/user_guide/brain.rst

```diff
@@ -989,45 +989,18 @@ demonstrated empirical value in many model training exercises.
 Managing brain runs
 ___________________
 
-When you run a brain method on a dataset, the run is recorded on the dataset,
-allowing you to retrive information about it later, delete it (along with any
-modifications to your dataset that were performed by it), or even retrieve the
-view into your dataset that you processed.
-
-Brain method runs can be accessed later by their `brain_key`:
-
-.. tabs::
-
-    .. tab:: Visualizations
```

Comment:
Why are we deleting all this? Could still be useful as an FYI

Reply:
I'll add it back.